desktop/apps/consignments2/components/checkbox_input/index.js

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import block from 'bem-cn'
+
+export const renderCheckboxInput = ({ input, ...custom }) => (
+  <CheckboxInput
+    value={input.value}
+    onChange={input.onChange}
+    {...custom}
```

Comment:
In general, I would always move spread operations above explicitly defined properties so that you can have more certainty that they won't be overwritten.

(I still need to grok overall architecture so consider these intermediate comments)